packages/playground/package.json

```diff
@@ -41,13 +41,13 @@
     "@material-ui/core": "^4.12.4",
     "@mui/material": "^5.14.8",
     "@rjsf/antd": "^5.14.2",
-    "@rjsf/bootstrap-4": "^5.14.2",
     "@rjsf/chakra-ui": "^5.14.2",
     "@rjsf/core": "^5.14.2",
     "@rjsf/fluent-ui": "^5.14.2",
     "@rjsf/fluentui-rc": "^5.14.2",
     "@rjsf/material-ui": "^5.14.2",
     "@rjsf/mui": "^5.14.2",
+    "@rjsf/react-bootstrap": "file:../react-bootstrap",
```

Comment:
This lets me build the playground without releasing the theme to NPM. We can update this to the version number once we release a v6 beta.